RocketChat/Rocket.Chat
# fix: New `custom-roles` license module isn't properly checked

## Proposed changes (including videos or screenshots)


 - Check the new `custom-roles` license module in order to enable the custom roles feature (on the server and client);
 - Update the "role deletion" modal's text to consider the new plans (this modal is displayed for Community, Starter and Pro plans before deleting a role).



## Issue(s)

Introduced by #30551

## Steps to test or reproduce

Go to **Workspace > Permission > New Role**, the behavior of this button and editing roles should follow what is defined for each license (this feature is only enabled for the Enterprise plan). The `roles.create` and `roles.update` endpoints should also be blocked for non-Enteprise workspaces.

## Further comments

NBJ-576
